src/model.rs

```diff
@@ -303,3 +316,48 @@ mod tests {
         )
     }
 }
+
+/// This is the main structure provided by this crate. It can be used
+/// for both weight training, and for reviews.
+pub struct Fsrs<B: Backend<FloatElem = f32> = NdArrayBackend> {
```

Comment:
It looks weird that only `F` is capital. FSRS is abbreviation of free spaced repetition scheduler. I recommend using `FSRS` here.